curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/create-a-media-query.english.md

```diff
@@ -26,10 +26,13 @@ Add a media query, so that the <code>p</code> tag has a <code>font-size</code> o
 
 ```yml
 tests:
-  - text: Your <code>p</code> element should have the <code>font-size</code> of 10px when the device <code>height</code> is less than or equal to 800px.
-    testString: assert($('p').css('font-size') == '10px', 'Your <code>p</code> element should have the <code>font-size</code> of 10px when the device <code>height</code> is less than or equal to 800px.');
   - text: Declare a <code>@media</code> query for devices with a <code>height</code> less than or equal to 800px.
-    testString: assert(code.match(/@media\s*?\(\s*?max-height\s*?:\s*?800px\s*?\)/g), 'Declare a <code>@media</code> query for devices with a <code>height</code> less than or equal to 800px.');
+    testString: assert($("style").text().replace(/\s/g ,'').match(/@media\(max-height:800px\)/g));
+  - text: Your <code>p</code> element should have the <code>font-size</code> of 10px when the device <code>height</code> is less than or equal to 800px.
+    testString: assert($("style").text().replace(/\s/g ,'').match(/@media\(max-height:800px\)\{p\{font-size:10px;\}\}/g));
```

Comment:
It is not necessary to escape { or }.  Also, you should make the semi-colon optional if there are no other properties after a property/value pair.